resources/views/functions/user_detail.blade.php

```diff
@@ -0,0 +1,132 @@
+                        </span> {{ $user->name }}
+                            </li>
+                            <li class="list-group-item text-right">
+                        <span class="pull-left">
+                            <strong class="">{{ trans('message.infor_user.join') }} &#58; </strong>
+                        </span>
+                                {{ date('d-m-Y', strtotime($user->created_at)) }}
+                            </li>
+                            <li class="list-group-item text-right">
+                        <span class="pull-left">
+                            <strong class="">{{ trans('message.infor_user.account') }} &#58; </strong>
+                        </span> {{ $user->username }}
+                            </li>
+                            <li class="list-group-item text-right">
+                        <span class="pull-left">
+                            <strong class="">{{ trans('message.infor_user.email') }} &#58; </strong>
+                        </span> {{ $user->email }}
+                            </li>
+                            <li class="list-group-item text-right">
+                        <span class="pull-left">
+                            <strong class="">{{ trans('message.infor_user.phoneNumber') }} &#58; </strong>
+                        </span> {{ $user->phone_number }}
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-8">
+                <h3> {{ trans('message.booking.yourBookingHistory') }}: </h3>
+                <table class="table">
+                    <thead class="thead-light">
+                    <tr>
+                        <th scope="col">#</th>
+                        <th scope="col">{{ trans('message.booking.bookingId') }}</th>
+                        <th scope="col">{{ trans('message.room') }}</th>
+                        <th scope="col">{{ trans('message.booking.checkin') }}</th>
+                        <th scope="col">{{ trans('message.booking.checkout') }}</th>
+                        <th scope="col">{{ trans('message.booking.timeBooking') }}</th>
+                        <th scope="col">{{ trans('message.functions.price') }}</th>
+                        <th scope="col">{{ trans('message.booking.status') }}</th>
+                        <th scope="col">{{ trans('message.booking.tools') }}</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    @if ($bookings->count())
+                        @foreach ($bookings as $key => $booking)
+                            <tr>
+                                <th scope="row">{{ $key + 1 }}</th>
+                                <td>{{ $booking->id }}</td>
+                                <td>@foreach ($booking->rooms as $room) {{ $room->name  }} @endforeach</td>
```

Comment:
nối tên dấu cách vào
```suggestion
                                <td>@foreach ($booking->rooms as $room) {{ $room->name . ','  }} @endforeach</td>
```